### ISSUE TITLE
not able to process GitHub schema file

### ISSUE THREAD

**Issue by houstonhaynes** [Zaid-Ajaj/Snowflaqe#26]
I'm hitting a few bumps in the road while trying to run it. The hitch seems to be with parsing the schema file. I've tried two examples. First was the one I gathered from the FSharp.Data.GraphQL sample. It had a commenting style that was triple-double-quoted above and below each line.

`Reading configuration from C:\repo\GitHubGraphQL\snowflaqe.json
⏳ Loading GraphQL schema from ./schema.docs.graphql
Syntax Error GraphQL (1:1) Unexpected String ""
1: """
   ^
2: Defines what type of global IDs are accepted for a mutation argument of type ID.`

Then I tried another that came from running the introspection query which gave back a json result. That too hit an error, but this time on a null. 

`Reading configuration from C:\repo\GitHubGraphQL\snowflaqe.json
⏳ Loading GraphQL schema from ./schema.docs.json
Value cannot be null. (Parameter 'source')`

So I'm stuck - to a first approximation. I'm continuing to work on the file handling for updating/mutating the CSV, and will save the best for last. 😁 I'm  attaching both files here. Thanks in advance for giving it a once-over when you can. 

[schema.docs.zip](https://github.com/Zaid-Ajaj/Snowflaqe/files/6053847/schema.docs.zip)

**Comment by Zaid-Ajaj** [comment]
Hi @houstonhaynes, thanks for the issue report. I will try to figure out why the introspection results aren't working 😓

**Comment by Zaid-Ajaj** [comment]
There is something funny going on with the schema file. I was just debugging the parsing logic. It does fail but it is because the schema is missing some crucial information about the objects and their types. I already have a sample Github schema that I run my tests against and it has this type for example:

This is a piece of my local schema.json file generated from Github's API
```json
 {
                    "kind": "INTERFACE",
                    "name": "Actor",
                    "description": "Represents an object which can take actions on GitHub. Typically a User or Bot.",
                    "fields": [
                        {
                            "name": "avatarUrl",
                            "description": "A URL pointing to the actor's public avatar.",
                            "args": [
                                {
                                    "name": "size",
                                    "description": "The size of the resulting square image.",
                                    "type": {
                                        "kind": "SCALAR",
                                        "name": "Int",
                                        "ofType": null
                                    },
                                    "defaultValue": null
                                }
                            ],
                            "type": {
                                "kind": "NON_NULL",
                                "name": null,
                                "ofType": {
                                    "kind": "SCALAR",
                                    "name": "URI",
                                    "ofType": null
                                }
                            },
                            "isDeprecated": false,
                            "deprecationReason": null
                        },
                        {
                            "name": "login",
                            "description": "The username of the actor.",
                            "args": [],
                            "type": {
                                "kind": "NON_NULL",
                                "name": null,
                                "ofType": {
                                    "kind": "SCALAR",
                                    "name": "String",
                                    "ofType": null
                                }
                            },
                            "isDeprecated": false,
                            "deprecationReason": null
                        },
                        {
                            "name": "resourcePath",
                            "description": "The HTTP path for this actor.",
                            "args": [],
                            "type": {
                                "kind": "NON_NULL",
                                "name": null,
                                "ofType": {
                                    "kind": "SCALAR",
                                    "name": "URI",
                                    "ofType": null
                                }
                            },
                            "isDeprecated": false,
                            "deprecationReason": null
                        },
                        {
                            "name": "url",
                            "description": "The HTTP URL for this actor.",
                            "args": [],
                            "type": {
                                "kind": "NON_NULL",
                                "name": null,
                                "ofType": {
                                    "kind": "SCALAR",
                                    "name": "URI",
                                    "ofType": null
                                }
                            },
                            "isDeprecated": false,
                            "deprecationReason": null
                        }
                    ],
                    "inputFields": null,
                    "interfaces": null,
                    "enumValues": null,
                    "possibleTypes": [
                        {
                            "kind": "OBJECT",
                            "name": "Bot",
                            "ofType": null
                        },
                        {
                            "kind": "OBJECT",
                            "name": "EnterpriseUserAccount",
                            "ofType": null
                        },
                        {
                            "kind": "OBJECT",
                            "name": "Mannequin",
                            "ofType": null
                        },
                        {
                            "kind": "OBJECT",
                            "name": "Organization",
                            "ofType": null
                        },
                        {
                            "kind": "OBJECT",
                            "name": "User",
                            "ofType": null
                        }
                    ]
                },
```
However, that same type from your introspection results is missing the types of the object fields
```json
                {
                    "name": "Actor",
                    "kind": "INTERFACE",
                    "description": "Represents an object which can take actions on GitHub. Typically a User or Bot.",
                    "fields": [{
                            "name": "avatarUrl"
                        },
                        {
                            "name": "login"
                        },
                        {
                            "name": "resourcePath"
                        },
                        {
                            "name": "url"
                        }
                    ]
                },
```
So the question is, how did you generate the JSON schema file? Did you use the [Standard Introspection Query](https://github.com/Zaid-Ajaj/Snowflaqe/blob/master/src/Introspection.gql) against the API?

**Comment by houstonhaynes** [comment]
So the short answer is yes. Just to make sure I pulled the one linked here and diffed it. The queries match. I just re-ran the introspection query and diffed the result against what I saved last night (local time) and it's WILDLY different. It's lit up like Christmas. SO OF COURSE running it this morning it worked without a hitch. Geez I feel like the person who takes their car to the shop to have the mechanic look at their car - and then it fails to make the bothersome noise that caused the concern in the first place... 🙄

![image](https://user-images.githubusercontent.com/8174976/109386458-91766f00-78af-11eb-8358-f305fbf70d8a.png)
